app/controllers/update_username_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_dependency "#{Rails.root}/lib/importers/user_importer"
+
+class UpdateUsernameController < ApplicationController
+  before_action :require_signed_in
+  respond_to :html, :json
+  def index
+    @user = User
+  end
+
+  def update
+    @username = params['username']
+    UserImporter.update_username_for_global_id(@username)
```

Comment:
Upon further reflection, I think this should not be calling `UserImporter.update_username_for_global_id` without sanitizing the user input first. I think that will require some refactoring of UserImporter, because the sanitization we want is in the first part of the .new_from_username method, lines 32 through 48. That could be extracted into its own method and used here.